xyz.judro.mines.yml

```diff
@@ -0,0 +1,22 @@
+app-id: xyz.judro.mines
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: mines
+rename-desktop-file: mines.desktop
+rename-appdata-file: mines.metainfo.xml
+rename-icon: mines
+modules:
+  - name: mines
+    buildsystem: simple
+    build-commands:
+      - install -D mines /app/bin/mines
```

Comment:
so you are copying a binary but building a flatapk for both x86_64 and aarch64. Why no building from source instead?